website/docs/r/vapp_vm.html.markdown

```diff
@@ -711,67 +723,167 @@ resource "vcd_vapp_vm" "web2" {
-  `win95Guest` (Microsoft Windows 95), `win31Guest` (Microsoft Windows 3.1), `dosGuest` (Microsoft MS-DOS), 
-  `winXPPro64Guest` (Microsoft Windows XP Professional (64-bit)), `winXPProGuest` (Microsoft Windows XP Professional (32-bit)), 
-  `winVista64Guest` (Microsoft Windows Vista (64-bit)), `winVistaGuest` (Microsoft Windows Vista (32-bit)), 
-  `windows7Server64Guest` (Microsoft Windows Server 2008 R2 (64-bit)), `winLonghorn64Guest` (Microsoft Windows Server 2008 (64-bit)), 
-  `winLonghornGuest` (Microsoft Windows Server 2008 (32-bit)), `winNetWebGuest` (Microsoft Windows Server 2003 Web Edition (32-bit)), 
-  `winNetStandard64Guest` (Microsoft Windows Server 2003 Standard (64-bit)), 
-  `winNetStandardGuest` (Microsoft Windows Server 2003 Standard (32-bit)), 
-  `winNetDatacenter64Guest` (Microsoft Windows Server 2003 Datacenter (64-bit)), 
-  `winNetDatacenterGuest` (Microsoft Windows Server 2003 Datacenter (32-bit)), `winNetEnterprise64Guest` (Microsoft Windows Server 2003 (64-bit)), 
-  `winNetEnterpriseGuest` (Microsoft Windows Server 2003 (32-bit)), `winNTGuest` (Microsoft Windows NT), 
-  `windows7_64Guest` (Microsoft Windows 7 (64-bit)), `windows7Guest` (Microsoft Windows 7 (32-bit)), `win2000ServGuest` (Microsoft Windows 2000 Server), 
-  `win2000ProGuest` (Microsoft Windows 2000 Professional), `win2000AdvServGuest` (Microsoft Windows 2000), 
-  `winNetBusinessGuest` (Microsoft Small Business Server 2003)
-* Other Os:
-  `freebsd12_64Guest` (FreeBSD 12 or later versions (64-bit)), `freebsd12Guest` (FreeBSD 12 or later versions (32-bit)), 
-  `freebsd11_64Guest` (FreeBSD 11 (64-bit)), `freebsd11Guest` (FreeBSD 11 (32-bit)), `darwin18_64Guest` (Apple macOS 10.14 (64-bit)), 
-  `darwin17_64Guest` (Apple macOS 10.13 (64-bit)), `darwin16_64Guest` (Apple macOS 10.12 (64-bit)), `darwin15_64Guest` (Apple Mac OS X 10.11 (64-bit)), 
-  `darwin14_64Guest` (Apple Mac OS X 10.10 (64-bit)), `darwin13_64Guest` (Apple Mac OS X 10.9 (64-bit)), 
-  `darwin12_64Guest` (Apple Mac OS X 10.8 (64-bit)), `eComStation2Guest` (Serenity Systems eComStation 2), `openServer6Guest` (SCO OpenServer 6), 
-  `solaris11_64Guest` (Oracle Solaris 11 (64-bit)), `darwin11_64Guest` (Apple Mac OS X 10.7 (64-bit)), 
-  `darwin11Guest` (Apple Mac OS X 10.7 (32-bit)), `eComStationGuest` (Serenity Systems eComStation 1), `unixWare7Guest` (SCO UnixWare 7), 
-  `openServer5Guest` (SCO OpenServer 5), `os2Guest` (IBM OS/2), `freebsd64Guest` (FreeBSD Pre-11 versions (64-bit)), 
-  `freebsdGuest` (FreeBSD Pre-11 versions (32-bit)), `darwin10_64Guest` (Apple Mac OS X 10.6 (64-bit)), 
-  `darwin10Guest` (Apple Mac OS X 10.6 (32-bit)), `otherGuest64` (Other (64-bit)), `otherGuest` (Other (32-bit)),
-   `solaris10_64Guest` (Oracle Solaris 10 (64-bit)), `solaris10Guest` (Oracle Solaris 10 (32-bit)), `netware6Guest` (Novell NetWare 6.x), 
-   `netware5Guest` (Novell NetWare 5.1)
+* **Linux**
+- `other5xLinux64Guest` - Other 5.x or later Linux (64-bit)
```

Comment:
Using `-` instead of `*` does not guarantee a change of bullet point. To do a good differentiation, we need to indent the other lines.
```
* **Linux**
  - `other5xLinux64Guest` - Other 5.x or later Linux (64-bit)
  - `other5xLinuxGuest` - Other 5.x or later Linux (32-bit)
```
Even using the same bullet will work, provided we have the indentation
```
* **Linux**
  * `other5xLinux64Guest` - Other 5.x or later Linux (64-bit)
  * `other5xLinuxGuest` - Other 5.x or later Linux (32-bit)
```

Both will look as below
<img width="555" alt="Screenshot 2023-08-28 at 16 21 43" src="https://github.com/vmware/terraform-provider-vcd/assets/11331818/a0a68967-1851-45f6-8562-66f9113a7bd6">

Reply:
Thanks for the suggestion, updated.